example/testapp/apitest/features/support/feature_hooks.js

```diff
@@ -2,14 +2,32 @@
 var cucumberApi = require('cucumber-api');
 var Promise = require('bluebird');
 var childProcess = Promise.promisifyAll(require('child_process'), {multiArgs: true});
+var portUsed = require('tcp-port-used');
+
+waitForStartup = function (done) {
+  portUsed.check(8080, "localhost")
+    .then(function(inUse) {
+      if(inUse) {
+        done();
+      }
+      else {
+        setTimeout(function() {
+          waitForStartup(done);
+        }, 1000);
```

Comment:
1000 is a long polling interval.  You could easily get away with 50, and have a zippier startup.